bin/notify-users-on-new-locations

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+"""
+Notifies specified parties of new data for their interested locations
```

Comment:
I used the word 'parties' previously in this script but later renamed it to 'users'. I should update the docstring here to reflect that change.